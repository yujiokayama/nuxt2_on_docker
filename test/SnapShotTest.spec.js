import { shallowMount } from '@vue/test-utils'
import SnapShotTest from '@/components/test/SnapShotTest.vue'

describe('コンポーネントのテスト', () => {
  const wrapper = shallowMount(SnapShotTest)
  it.only('正しくレンダリングされるか', () => {
    expect(wrapper.element).toMatchSnapshot()
  })

  it('messageが正しいか', () => {
    // wrapperのテキストは「Vue Test Utilsを使ったコンポーネントのテスト」である
    expect(wrapper.text()).toBe('Vue Test Utilsを使ったコンポーネントのテスト')
  })
})
