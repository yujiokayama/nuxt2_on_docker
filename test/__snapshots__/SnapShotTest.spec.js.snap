// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`コンポーネントのテスト 正しくレンダリングされるか 1`] = `
<div>
  <p>
    Vue Test Utilsを使ったコンポーネントのテスト
  </p>
   
  <p>
    Vue Test Utilsを使ったコンポーネントのテスト
  </p>
</div>
`;
